src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -642,15 +645,48 @@ public ResponseEntity<Void> moveOrCopyModifications(@PathVariable("studyUuid") U
     @PutMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/loadflow/run")
     @Operation(summary = "run loadflow on study")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The loadflow has started")})
-    public ResponseEntity<Void> runLoadFlow(
+    public ResponseEntity<UUID> runLoadFlow(
             @PathVariable("studyUuid") UUID studyUuid,
-            @PathVariable("nodeUuid") UUID nodeUuid) {
+            @PathVariable("nodeUuid") UUID nodeUuid,
+            @RequestHeader(HEADER_USER_ID) String userId) {
         studyService.assertIsNodeNotReadOnly(nodeUuid);
-        studyService.assertLoadFlowRunnable(nodeUuid);
-        studyService.runLoadFlow(studyUuid, nodeUuid);
+        return ResponseEntity.ok().body(studyService.runLoadFlow(studyUuid, nodeUuid, userId));
+    }
+
+    @GetMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/loadflow/result")
+    @Operation(summary = "Get a loadflow result on study")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The loadflow result"),
+            @ApiResponse(responseCode = "204", description = "No loadflow has been done yet"),
+            @ApiResponse(responseCode = "404", description = "The loadflow has not been found")})
+    public ResponseEntity<String> getLoadflowResult(@Parameter(description = "study UUID") @PathVariable("studyUuid") UUID studyUuid,
+                                                        @Parameter(description = "nodeUuid") @PathVariable("nodeUuid") UUID nodeUuid) {
+        String result = loadflowService.getLoadFlowResult(nodeUuid);
+        return result != null ? ResponseEntity.ok().body(result) :
+                ResponseEntity.noContent().build();
+    }
+
+    @GetMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/loadflow/status")
+    @Operation(summary = "Get the loadflow status on study")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The loadflow status"),
+            @ApiResponse(responseCode = "204", description = "No loadflow has been done yet"),
+            @ApiResponse(responseCode = "404", description = "The loadflow status has not been found")})
```

Comment:
```suggestion
            @ApiResponse(responseCode = "404", description = "The loadflow has not been found")})
```
To be coherent with above